packages/react/ds/tailwind.config.ts

```diff
@@ -3,7 +3,7 @@ import { Config } from 'tailwindcss';
 
 const config: Config = {
   prefix: 'gi-',
-  content: ['./src/**/*.tsx'],
+  content: ['../ds/src/**/*.tsx'],
```

Comment:
This as well?

@BogdanDanilescu

Reply:
no ,we can remove this change. this was for the tailwing config in the storybook folder. it tells tailwind where to look for classes in order to know which classes should be built at runtime